crates/download/src/main.rs

```diff
@@ -53,15 +53,15 @@ struct Args {
     min: bool,
 
     /// 出力先の指定
-    #[arg(short, long, default_value(DEFAULT_OUTPUT))]
+    #[arg(short, long, value_name("DIRECTORY"), default_value(DEFAULT_OUTPUT))]
     output: PathBuf,
 
     /// ダウンロードするvoicevox_coreのバージョンの指定
-    #[arg(short, long, default_value("latest"))]
+    #[arg(short, long, value_name("GIT_TAG_OR_LATEST"), default_value("latest"))]
```

Comment:
ニュアンス的にこっちの方があってそうかも？
```suggestion
    #[arg(short, long, value_name("GITHUB_TAG_OR_LATEST"), default_value("latest"))]
```

Reply:
変数名で`GIT_TAG_OR_LATEST`が使われてるのを確認しました。どちらでも良いのかなという気持ちです。

Reply:
なるほどです。まあそこまでこだわりないので良さそう！